deepmd/pt/loss/ener_hess.py

```diff
@@ -0,0 +1,454 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+from typing import (
+    Optional,
+)
+
+import torch
+import torch.nn.functional as F
+
+from deepmd.pt.loss.loss import (
+    TaskLoss,
+)
+from deepmd.pt.utils import (
+    env,
+)
+from deepmd.pt.utils.env import (
+    GLOBAL_PT_FLOAT_PRECISION,
+)
+from deepmd.utils.data import (
+    DataRequirementItem,
+)
+
+
+class EnergyHessianStdLoss(TaskLoss):
```

Comment:
It looks to me that the energy loss should be reused rather than being rewritten.

Reply:
You are right. I think this confusion is solved by merging `ener_hess.py` to `ener.py`, where class `EnergyHessianStdLoss`  inherits from `EnergyStdLoss`.